### DISCUSSION TITLE
Account Suspended

### DISCUSSION THREAD

**Discussion by Thaitheya** [community/community#144244] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Other

### Reg: My account suspension
My account has been suspended without any notice and I have more works to be committed but it's very difficult to handle it on my drive and computer. So my account to be freely accessible.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Thaitheya, 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

_Please note: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
